mesh_map/include/mesh_map/mesh_map.h

```diff
@@ -424,35 +434,23 @@ class MeshMap
   //! global frame / coordinate system id
   std::string global_frame;
 
-  //! server url
-  std::string srv_url;
-
-  //! login username to connect to the server
-  std::string srv_username;
-
-  //! login password to connect to the server
-  std::string srv_password;
-
-  std::string mesh_layer;
-
-  double min_roughness;
-  double max_roughness;
-  double min_height_diff;
-  double max_height_diff;
-  double bb_min_x;
-  double bb_min_y;
-  double bb_min_z;
-  double bb_max_x;
-  double bb_max_y;
-  double bb_max_z;
-
-
+  //! mesh file 
   std::string mesh_file;
+
+  //! mesh part
   std::string mesh_part;
+  
+  //! mesh working file
+  std::string mesh_working_file;
+
+  //! mesh working part
+  std::string mesh_working_part;
```

Comment:
The comments are superfluous, since they only consist of the variable's name. Leaving them in place does not add any clarity and they might become out-of-date when the code changes, leading to confusion.
Either remove them or replace them by an explanation, if necessary.

Reply:
Also, you could add a `path` or `file_path` to the variable name, if it is a path. I think e.g. `mesh_file` is probably **not** the actual file (or pointer to some data in that file), but just the filesystem path. Since its type is `std::string`! :) But that might just be my personal preference. :shrug:

Reply:
What `working` means could use an explanation.
Why are there working and non-working versions of file&part?

Maybe salvage a few words from your PR description:
Internally it is divided into an input file and working file. The input file is an arbitrary mesh format (that can be loaded by assimp or HDF5) the working file is always a HDF5. I added a service which can be used to save the current state to the working file.

Or maybe find better names, e.g. call them `mesh_input_file`/`mesh_input_part` and `mesh_working_file`/`mesh_working_part`.

Reply:
> The comments are superfluous, since they only consist of the variable's name. Leaving them in place does not add any clarity and they might become out-of-date when the code changes, leading to confusion. Either remove them or replace them by an explanation, if necessary.

Agree. I was just adapting to the comment style that was already there :grin: There could be a complete PR about cleaning up all of the comments plus all the error messages.

Reply:
> What `working` means could use an explanation. Why are there working and non-working versions of file&part?
> 
> Maybe salvage a few words from your PR description: Internally it is divided into an input file and working file. The input file is an arbitrary mesh format (that can be loaded by assimp or HDF5) the working file is always a HDF5. I added a service which can be used to save the current state to the working file.
> 
> Or maybe find better names, e.g. call them `mesh_input_file`/`mesh_input_part` and `mesh_working_file`/`mesh_working_part`.

Agree. I will push a commit with some docs soon.

Reply:
Great, thanks!